intrastat_product/models/intrastat_product_declaration.py

```diff
@@ -490,8 +490,30 @@ def _get_incoterm(self, inv_line, notedict):
         return incoterm
 
     def _get_product_origin_country(self, inv_line, notedict):
+        _logger.warning(
```

Comment:
In Odoo Community, they are using warnings.warn with a DeprecationWarning and this warning is shown in logs but also in the IDE (at least Pycharm does it) as Deprecated https://github.com/odoo/odoo/blob/687d88eeea836aacbcd1b079e82e597d1bd9309c/addons/account/models/account_move.py#L119